### PR TITLE
Feature/rework the way we handle the build path

### PR DESCRIPTION
**Pull request by nikomall34**
Basically what I did here is I check in the `ReportPathGeneratorService.getReportMutationsFile` if the debug buildtype exists. If it doesn't I just use the path that we used before. This way the Annotator also displays the mutations results for the android project.

### CODE REVIEW

**Review by lheimbs — CHANGES_REQUESTED** [review]
Maybe remove the code that is already present in other PRs and not really needed for this.

Also since the coverage file's path is the same maybe put it into the getReportPath so that the coverage file's path can be used from here too

**Comment by nikomall34** [comment]
Luca is doing this in his ticket. Should I still add it?

**Comment by lheimbs** [comment]
I'm not saying load the coverage report. but change the path for both files not only for the mutations since both will be in the same folder anyways

File: pitmutationmate/src/main/kotlin/com/amos/pitmutationmate/pitmutationmate/services/ReportPathGeneratorService.kt [hunk 1]
```diff
@@ -27,12 +29,20 @@ class ReportPathGeneratorService(private val project: Project) {
         return Path.of("$projectBasePath/build/reports/pitest/test")
     }
 
+    private fun checkForDebugBuiltType(): Path {
+        var path = getReportPath()
+        if (Files.exists(Paths.get("$path/debug"))) {
```
Comment (lheimbs) [inline pitmutationmate/src/main/kotlin/com/amos/pitmutationmate/pitmutationmate/services/ReportPathGeneratorService.kt:34]:
```suggestion
        if (Files.exists(Path.of("$path/debug"))) {
```
Sorry for the pickyness but lets keep it consistent :see_no_evil:

**Review by lheimbs — APPROVED** [review]

**Pull request merged**